tensorflow/lite/micro/kernels/detection_postprocess_test.cc

```diff
@@ -0,0 +1,482 @@
+  0.0, 0.0,  0.0, 0.0,  // box #1
+  0.0, 1.0,  0.0, 0.0,  // box #2
+  0.0, -1.0, 0.0, 0.0,  // box #3
+  0.0, 0.0,  0.0, 0.0,  // box #4
+  0.0, 1.0,  0.0, 0.0,  // box #5
+  0.0, 0.0,  0.0, 0.0   // box #6
+};
+
+// class scores - two classes with background
+static const float kInputData2[] = {0., .9, .8, 0., .75, .72, 0., .6, .5, 0., .93, .95, 0.,
+                             .5, .4, 0., .3, .2};
+
+// six anchors in center-size encoding
+static const float kInputData3[] = {
+  0.5, 0.5,   1.0, 1.0,  // anchor #1
+  0.5, 0.5,   1.0, 1.0,  // anchor #2
+  0.5, 0.5,   1.0, 1.0,  // anchor #3
+  0.5, 10.5,  1.0, 1.0,  // anchor #4
+  0.5, 10.5,  1.0, 1.0,  // anchor #5
+  0.5, 100.5, 1.0, 1.0   // anchor #6
+};
+// Same boxes in box-corner encoding:
+// { 0.0, 0.0, 1.0, 1.0,
+//   0.0, 0.1, 1.0, 1.1,
+//   0.0, -0.1, 1.0, 0.9,
+//   0.0, 10.0, 1.0, 11.0,
+//   0.0, 10.1, 1.0, 11.1,
+//   0.0, 100.0, 1.0, 101.0}
+
+static const float kGolden1[] = {0.0, 10.0, 1.0, 11.0, 0.0, 0.0, 1.0, 1.0, 0.0, 100.0, 1.0, 101.0};
+static const float kGolden2[] = {1, 0, 0};
+static const float kGolden3[] = {0.95, 0.9, 0.3};
+static const float kGolden4[] = {3.0};
+
+
+void TestDetectionPostprocess(const int* input_dims_data1, const float* input_data1,
+                              const int* input_dims_data2, const float* input_data2,
+                              const int* input_dims_data3, const float* input_data3,
+                              const int* output_dims_data1, float* output_data1,
+                              const int* output_dims_data2, float* output_data2,
+                              const int* output_dims_data3, float* output_data3,
+                              const int* output_dims_data4, float* output_data4,
+                              const float* golden1, const float* golden2,
+                              const float* golden3, const float* golden4,
+                              const float tolerance, bool use_regular_nms,
+                              uint8_t* input_data_quantized1=nullptr,
+                              uint8_t* input_data_quantized2=nullptr,
+                              uint8_t* input_data_quantized3=nullptr,
+                              const float input_min1=0, const float input_max1=0,
+                              const float input_min2=0, const float input_max2=0,
+                              const float input_min3=0, const float input_max3=0) {
+  TfLiteIntArray* input_dims1 = IntArrayFromInts(input_dims_data1);
+  TfLiteIntArray* input_dims2 = IntArrayFromInts(input_dims_data2);
+  TfLiteIntArray* input_dims3 = IntArrayFromInts(input_dims_data3);
+  TfLiteIntArray* output_dims1 = nullptr;
+  TfLiteIntArray* output_dims2 = nullptr;
+  TfLiteIntArray* output_dims3 = nullptr;
+  TfLiteIntArray* output_dims4 = nullptr;
+
+  // Instance of a zero-length int to pass as tensor dims for a flatbuffer
+  // Tensor with no shape.
+  const TfLiteIntArray kZeroLengthIntArray = {0};
```

Comment:
Please create int array and use IntArrayFromInts as follows:
const int zero_length_int_array_data[] = {0};
TfLiteIntArray zero_length_int_array = IntArrayFromInts(zero_length_int_array_data);

Reply:
Done